LICENSE

```diff
@@ -1,5 +1,5 @@
 The MIT License (MIT)
-Copyright (c) 2019 Aldwin Vlasblom
+Copyright (c) 2021 Aldwin Vlasblom
```

Comment:
Does this not happen automatically during the release process?

Reply:
It does, but this is an upstream (from the template repo) change that got included automatically upon merging. So let's say it gets updated in one of two ways. :)